### PR TITLE
Complete registerMethod2 migration

### PR DESCRIPTION
**Pull request by pentacular**

### CODE REVIEW

File: algorithm/cgal/FromPolygonSoup.h [hunk 1]
```diff
@@ -5,55 +5,75 @@ int FromPolygonSoup(Geometry* geometry, emscripten::val fill, bool wrap_always,
...
+      std::cout << "QQ/FromPolygonSoup/Wrap: alpha=" << alpha
+                << " offset=" << offset << std::endl;
+      std::cout << "QQ/FromPolygonSoup/Wrap/MeshClear" << std::endl;
+      mesh.clear();
+      std::cout << "QQ/FromPolygonSoup/Wrap/Wrap" << std::endl;
+      wrap_epick(points, faces, alpha, offset, mesh);
+    }
 
-  demesh(mesh);
+    if (corner_threshold > 0) {
```
Comment (BarbourSmith) [inline algorithm/cgal/FromPolygonSoup.h:66]: This is a nice way to do it. No need for a simplification flag and a corner threshold value

File: ui/app/App.js [hunk 2]
```diff
@@ -497,6 +497,18 @@ class App extends React.Component {
     };
 
     this.Notebook.run = async (path, options) => {
+      if (this.Notebook.runDebounce) {
```
Comment (BarbourSmith) [inline ui/app/App.js:500]: What are we debouncing? I think of that as being a hardware thing when a button is noisy

**Review by BarbourSmith — APPROVED** [review]
Congratulations on getting it all done :)

File: ui/app/App.js (hunk 2 above)
Comment (pentacular) [reply ui/app/App.js:500]: Same problem, I think -- something is triggering two simultaneous runs, but I haven't managed to track it down yet.

**Pull request merged**